### PR TITLE
pnmmargin, pnmmontage, pnmnlfilt, pnmnorm, pnmpad, pnmpaste: add pages

### PR DESCRIPTION
**Pull request by gutjuri**

### CODE REVIEW

File: pages/common/pnmmontage.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pnmmontage
+
+> Create a montage from multiple PNM images.
+> More information: <https://netpbm.sourceforge.net/doc/pnmmontage.html>.
+
+- Produce a packing of the specified images:
+
+`pnmmontage {{path/to/image1.pnm path/to/image2.pnm ...}} > {{path/to/output.pnm}}`
+
+- Specify the quality of the packing. Larger values produce smaller packings but take longer to compute:
```
Comment (kbdharun) [inline pages/common/pnmmontage.md:10]:
```suggestion
- Specify the quality of the packing (Note: larger values produce smaller packings but take longer to compute.):
```
Maybe, make it a note?

File: pages/common/pnmnorm.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# pnmnorm
+
+> Normalize the contrast in a PNM image.
+> See also: `pnmhisteq`.
+> More information: <https://netpbm.sourceforge.net/doc/pnmnorm.html>.
+
+- Force the brightest pixels to be white, the darkest pixels to be black and spread out the ones inbetween linearly:
```
Comment (kbdharun) [inline pages/common/pnmnorm.md:7]:
```suggestion
- Force the brightest pixels to be white, the darkest pixels to be black and spread out the ones in between linearly:
```

File: pages/common/pnmnlfilt.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# pnmnlfilt
+
+> Apply a nonlinear filter onto a PNM image.
```
Comment (kbdharun) [inline pages/common/pnmnlfilt.md:3]:
```suggestion
> Apply a non-linear filter onto a PNM image.
```

File: pages/common/pnmnorm.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`pnmnorm {{path/to/image.pnm}} > {{path/to/output.pnm}}`
+
+- Force the brightest pixels to be white, the darkest pixels to be black and spread out the ones inbetween quadratically such that pixels with a brightness of `n` become 50 percent bright:
```
Comment (kbdharun) [inline pages/common/pnmnorm.md:11]:
```suggestion
- Force the brightest pixels to be white, the darkest pixels to be black and spread out the ones in between quadratically such that pixels with a brightness of `n` become 50 % bright:
```
Maybe we can go with the symbol here as the line is already long, and could probably cause certain translations to cross the 256-character limit.

File: pages/common/pnmpad.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# pnmpad
+
+> Add borders to a PNM image.
+> See also: `pnmmargin`, `pamcut`, `pamcomp`.
+> More information: <https://netpbm.sourceforge.net/doc/pnmpad.html>.
+
+- Add borders of the specified sizes to the image:
+
+`pnmpad -left {{100}} -right {{150}} -top {{123} -bottom {{456}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```
Comment (kbdharun) [inline pages/common/pnmpad.md:9]:
```suggestion
`pnmpad -left {{100}} -right {{150}} -top {{123}} -bottom {{456}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```

File: pages/common/pnmpaste.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# pnmpaste
+
+> Paste a PNM image into another PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/pnmpaste.html>.
+
+- Paste a PNM image into another PNM image at the specified coordinates:
+
+`pnmpaste {{x}} {{y}} {{path/to/image1.pnm}} {{path/to/image2.pnm}} > {{path/to/output.pnm}}`
+
+- Paste the image read from stdin into the specified image:
```
Comment (kbdharun) [inline pages/common/pnmpaste.md:10]:
```suggestion
- Paste the image read from `stdin` into the specified image:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/common/pnmnlfilt.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# pnmnlfilt
+
+> Apply a non-linear filter onto a PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/pnmnlfilt.html>.
+
+- Apply the "alpha trimmed mean" filter with the specified alpha and radius values onto the PNM image:
+
+`pnmnlfilt {{0.0-0.5}} {{radius}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```
Comment (vitorhcl) [inline pages/common/pnmnlfilt.md:8]:
```suggestion
`pnmnlfilt {{0.0..0.5}} {{radius}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```

File: pages/common/pnmnlfilt.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+- Apply the "optimal estimation smoothing" filter with the specified noise threshold and radius onto the PNM image:
+
+`pnmnlfilt {{1.0-2.0}} {{radius}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```
Comment (vitorhcl) [inline pages/common/pnmnlfilt.md:12]:
```suggestion
`pnmnlfilt {{1.0..2.0}} {{radius}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```

File: pages/common/pnmnlfilt.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+- Apply the "edge enhancement" filter with the specified alpha and radius onto the PNM image:
+
+`pnmnlfilt {{-0.9-(-0.1)}} {{radius}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```
Comment (vitorhcl) [inline pages/common/pnmnlfilt.md:16]:
```suggestion
`pnmnlfilt {{-0.9..(-0.1)}} {{radius}} {{path/to/image.pnm}} > {{path/to/output.pnm}}`
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]
Thanks for your contribution. I suggest fixing the range placeholders to follow the style guide

**Review by vitorhcl — APPROVED** [review]

**Pull request merged**